torch_geometric/loader/link_neighbor_loader.py

```diff
@@ -291,6 +342,11 @@ def __init__(
         if edge_label is None:
             edge_label = torch.zeros(edge_label_index.size(1),
                                      device=edge_label_index.device)
+        self.edge_label = edge_label
+        if edge_time is not None and time_attr is None:
```

Comment:
Maybe in this case we could assume that the edge times are all larger than node times?

Reply:
We don't do temporal sampling if `time_attr` is `None`. So this if condition raises a warning now.